katas/8kyu/calculate-average.js

```diff
@@ -0,0 +1,7 @@
+function findAverage(array) {
+    if (array.length === 0) {
```

Comment:
I'd recommend using early exit pattern to avoid additional braces and depth.
I'll make the code more readable and clear.
Also, try to check false statement for length, because it includes `0` as well.
Examples:
```js
if (!array.length) {
  return 0;
}

return array.reduce((acc, current) => acc + current, 0) / array.length;
```